### DISCUSSION TITLE
Slack auth problem

### DISCUSSION THREAD

**Discussion by Comandante1977** [outline/outline#3671] [Self Hosting]
Hi, everybody!

Another problem with my self-hosting installation. 
Everything seems to begin working, but I've got the Slack auth error.
The first page ok:
![image](https://user-images.githubusercontent.com/45628795/174445174-6bf4d72b-640a-421e-b838-c96695d493d6.png)
And then:
![image](https://user-images.githubusercontent.com/45628795/174445199-35ffbb06-be7a-4df1-9310-9be29d4e355e.png)

All credentials are in place in .env file.
I'm not Slack admin, so may be admin did smth wrong.

In .env I have these lines configured:

https://outline.yourdomain.com/auth/slack.callback
SLACK_KEY=Slack Client ID value
SLACK_SECRET=Slakc Client Secret
SLACK_VERIFICATION_TOKEN=Slack Verification Token
SLACK_APP_ID=Slack App ID

Any ideas?

**Comment by tadopionero** [comment]
same isue

**Reply by bruno-offertech** [reply]
Same issue here....
